docs/guidelines/development.rst

```diff
@@ -7,15 +7,68 @@ This documents proposes development guidelines to increase work efficiency and s
 
 **NOTE**: All the proposal here are the results of authors' personal experiences. Saying that, if you have any idea to make them better you are very wellcome to create a PR.
 
+Overview
+=========
 
-1. Use *pre-commit* for formatting and linting
-===============================================
+1. `Making changes to the code-base by submitting a Pull Request <#>`_
+
+2. `Using *pre-commit* <#use-pre-commit-for-formatting-and-linting>`_
+
+
+Code Changes and Pull Request Submissions
+==========================================
+
+To make a change of the existing code base Pull Request (GitHub) or Merge Request (GitLab) are used.
```

Comment:
To make a change of the existing code base, Pull Request (GitHub) or Merge Request (GitLab) are used.

ADDING COMMA WILL BE MORE CLEAR